### PR TITLE
Modifications to figure in Appendix

### PR DESCRIPTION
**Pull request by pllim**
Partially address #27  (this PR does not change the caption)

The `from matplotlib ...` is more of a personal preference here. See https://stackoverflow.com/questions/11469336/what-is-the-difference-between-pylab-and-pyplot .

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -323,7 +323,7 @@ \section{Example}
     from astropy import coordinates, units as u, wcs
     from astroquery.skyview import SkyView
     from astroquery.vizier import Vizier
-    import pylab as pl
+    from matplotlib import pyplot as pl
```
Comment (keflavich) [inline main.tex]: :+1: to this

File: main.tex [hunk 2]
```diff
@@ -341,7 +341,7 @@ \section{Example}
     # use astropy's wcsaxes tool to create an RA/Dec image
     ax = fig.add_axes([0.15, 0.1, 0.8, 0.8], projection=mywcs)
     ax.set_xlabel("RA")
-    ax.set_ylabel("Dec")
+    ax.set_ylabel("DEC")
```
Comment (keflavich) [inline main.tex]: :-1: to this.  RA is capitalized because it's an acronym.  Dec is not capitalized because it's an abbreviation.

Comment (keflavich) [reply main.tex]: (so could you revert this change, then I'll merge the PR?)

**Review by keflavich — CHANGES_REQUESTED** [review]

File: main.tex (hunk 1 above)
Comment (bsipocz) [reply main.tex]: Super nitpicking, but according to astropy coding guidelines, this should be `import matplotlib.pyplot as plt`

**Comment by pllim** [comment]
Using GitHub's web interface to resolve conflict created a merge commit. I'll squash and re-push. Hold one.

**Comment by pllim** [comment]
Okay. I have addressed all the comments. Since we want to be consistent with Astropy snippets, I also changed `coordinates` import to `as coords`. Thank you all for the reviews!

**Comment by keflavich** [comment]
I'm OK with merge commits showing up in the history.

But, yes, this looks good to go.  I still stick to `import pylab as pl` (or `import matplotlib.pyplot as pl`), but I realize I'm now in the minority...

**Pull request merged**